### ISSUE TITLE
Right aligned pFrozenColumn and pTemplate="emptymessage" causing table to not render correctly

### ISSUE THREAD

**Issue by yuequinnlin** [primefaces/primeng#10894]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
Please demonstrate your case at stackblitz by using the issue template below. Issues without a test case have much less possibility to be reviewed in detail and assisted.

https://stackblitz.com/edit/primeng-tablescroll-demo-xgg6xh?file=src/app/app.component.html

**Current behavior**

When there is a frozen column that is not on the first column and there is an emptymessage pTemplate, getting the following error in console:
ERROR TypeError: Cannot read properties of undefined (reading 'style')
    at FrozenColumn.updateStickyPosition (primeng-tablescroll-demo-xgg6xh.stackblitz.io/turbo_modules/primeng@12.1.0-rc.1/bundles/primeng-table.umd.js:2562)

When I do a search filter that returns no results, after clearing the search input I get the same error and only one row is rendered.

**Expected behavior**

After search filtering, when there are no results and emptymessage is displayed, the table should render completely when search is cleared.

**Minimal reproduction of the problem with instructions**

https://stackblitz.com/edit/primeng-tablescroll-demo-xgg6xh?file=src/app/app.component.html
1. Navigate to above stackblitz.
2. Search for something random so it can display the emptymessage "No records found".
3. Clear the search results.
4. Only one row is rendered and there is error in console log.

* **Angular version:** 12.0.5

* **PrimeNG version:** 12.1.0-rc.1

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

**Comment by yigitfindikli** [comment]
Fixed in 12.2.3 and 13.x.x. https://github.com/primefaces/primeng/issues/10794

**Comment by yuequinnlin** [comment]
@yigitfindikli version 12.2.3 does not exist.  Please advise.  Thank you.
